test/test_standard_pipelines.py

```diff
@@ -167,6 +167,15 @@ def test_join_document_pipeline(document_store_dot_product_with_docs, reader):
     results = p.run(query=query)
     assert len(results["documents"]) == 3
 
+    # test concatenate with top_k_join parameter
+    join_node = JoinDocuments(join_mode="concatenate")
+    p = Pipeline()
+    p.add_node(component=es, name="R1", inputs=["Query"])
+    p.add_node(component=dpr, name="R2", inputs=["Query"])
+    p.add_node(component=join_node, name="Join", inputs=["R1", "R2"])
+    results = p.run(query=query, params={ 'Join': { 'top_k_join': 1 } })
+    assert len(results["documents"]) == 1
```

Comment:
We could also check that running the pipeline a second time with a different `top_k_join` parameter actually changes the number of documents returned. Would you add that?